cdc/sink/mq/manager/kafka_manager.go

```diff
@@ -142,7 +142,7 @@ func (m *kafkaTopicManager) getMetadataOfTopics() ([]*sarama.TopicMetadata, erro
 	topicMetaList, err := m.admin.DescribeTopics(topicList)
 	if err != nil {
 		log.Warn(
-			"Kafka admin client describe topics failed",
+			"kafka: get topics meta failed",
```

Comment:
If this log is supposed to be modified, line 153 should also be changed.

Reply:
I don't think there is a reason to change this. But it's Ok.
By the way, usually, the log is a sentence, so it should be capitalized at the beginning.

Reply:
I've revert the log. 

I think the log is not informative at the first glance, because `describe` is not a intuitive verb.

In the whole code base, most of the log is not capitalized as a sentence, I would prefer not to capitalize it to unify the convention.